src/Form/Form.js

```diff
@@ -734,18 +738,31 @@ class Form {
         return (!this.touched.has(input) && !input.classList.contains('ddg-autofilled'))
     }
 
+    /**
+     * Skip overridding values that the user provided if:
+     * - we're autofilling non credit card type and,
+     * - it's a previously filled input or,
+     * - it's a select input that was already "touched" by the user.
+     *  */
+    shouldSkipInput (input, dataType) {
```

Comment:
Can we add types to this method, there should be some examples nearby to copy - and also check the formatting of the final line of the comment :)